src/Styleguide/Components/Helpers.tsx

```diff
@@ -0,0 +1,16 @@
+/**
+ * Helper function to filter out blank and duplicate location names
+ * @param locations collection of location objects to filter
+ */
+export const filterLocations = (locations: ReadonlyArray<{ city: string }>) => {
+  if (!locations || locations.length < 1) {
+    return
+  }
+  const locationCities = locations.map((location, index) => {
+    return location.city
+  })
+  const filteredForDuplicatesAndBlanks = locationCities.filter((city, pos) => {
+    return city && locationCities.indexOf(city) === pos && city.length > 0
+  })
+  return filteredForDuplicatesAndBlanks
+}
```

Comment:
Helper files like this can be moved to the app that specifically uses it and named appropriately so that its easier to browse the file tree and see whats going on. A common idiom is 

`/Apps/Artworks/Utils/filterLocations.tsx`

Reply:
Can you also add a test for this under `/Apps/Artworks/Utils/__tests__/filterLocations.test.tsx`?